src/theme/constants.js

```diff
@@ -5,11 +5,11 @@ const constants = {
     large: "4px"
   },
   breakpoints: {
-    xSmall: "(min-width: 1px)",
-    small: "(min-width: 321px)",
+    small: "(max-width: 767px)",
     medium: "(min-width: 768px)",
     large: "(min-width: 1024px)",
     xLarge: "(min-width: 1440px)",
+    xSmallAndDown: "(min-width: 320px)",
```

Comment:
Should this be a max-width mq?

Reply:
I think either one works

Reply:
Semantically I think it should be `xSmallAndUp`

Reply:
took a look and you are right this should be `max-width`. Updated & I also changed the catalog sizes so we could debug xs display issues.